xpdschema/schemas/run_start.json

```diff
@@ -3,7 +3,8 @@
   "properties":{
     "facility": {
       "type": "string",
-      "description": "Facility Name (NSLS-II, APS, etc)"
+      "description": "Facility Name (NSLS-II, APS, etc)",
+      "enum": ["NSLS-II", "APS", "LCLS", "ALS", "CHESS"]
```

Comment:
SNS, ISIS, DIAMOND, DESEY, SPRING-8, ALBA, SOLEIL, ESRF, SLS, CLS, SSRL, and more